Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -578,20 +580,54 @@ public static string CreateDynamicRunSetJSON(Solution solution, RunsetExecutor r
                             mailReportConfig.MailSettings.SmtpDetails.User = runsetMailReport.Email.SMTPUser;
                             mailReportConfig.MailSettings.SmtpDetails.Password = runsetMailReport.Email.SMTPPass;
                         }
-                    }
+                    }                   
                     mailReportConfig.MailSettings.MailFrom = runsetMailReport.MailFrom;
                     mailReportConfig.MailSettings.MailTo = runsetMailReport.MailTo;
                     mailReportConfig.MailSettings.MailCC = runsetMailReport.MailCC;
                     mailReportConfig.MailSettings.Subject = runsetMailReport.Subject;
-
-                    mailReportConfig.Comments = runsetMailReport.Comments;
-                    if (runsetMailReport.EmailAttachments.Where(x => x.AttachmentType == EmailAttachment.eAttachmentType.Report).FirstOrDefault() != null)
+                    //Report mail content
+                    mailReportConfig.Comments = runsetMailReport.Comments;                   
+                    switch (runsetMailReport.HTMLReportTemplate)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace this 'switch' statement with 'if' statements to increase readability.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5700269)